frontends/web/src/routes/buy/exchange.tsx

```diff
@@ -175,6 +177,17 @@ export const Exchange = ({ code, accounts }: TProps) => {
   const cardFee = infoFeesDetail && infoFeesDetail.find(feeDetail => feeDetail.payment === 'card')?.fee;
   const bankTransferFee = infoFeesDetail && infoFeesDetail.find(feeDetail => feeDetail.payment === 'bank-transfer')?.fee;
 
+  const handleGoBack = () => {
+    if (accounts.length > 1) {
```

Comment:
somehow it didn't back 🤔 , looking at this, I  think `accounts.length === 1` is what was intended.

Reply:
> somehow it didn't back 

Hm... 🤔  is it possible that for some reason you didn't have any history in that tab (maybe you went directly to the URL)? In real life scenario I assume we wouldn't have to worry about this since the user is in the app (and URL isn't accessible).

Works as intended for me:

https://user-images.githubusercontent.com/28676406/222038608-dfe1db7c-cbb9-4613-8118-b33071ea7c14.mov

> looking at this, I think accounts.length === 1 is what was intended.

If I understood it correctly, when `accounts.length === 1` , we should actually go back twice. Because going back once will bring us to `buy/info` which will auto redirect us again to the selector page (when there's only one account). This case no "back" action will be visible. 

This's essentially the bug we're trying to fix.

Reply:
> is it possible that for some reason you didn't have any history in that tab (maybe you went directly to the URL)? 

yep maybe let me test again...